addons/web/static/tests/fields/relational_fields/field_many2one_tests.js

```diff
@@ -1115,6 +1115,41 @@ QUnit.module('fields', {}, function () {
             form.destroy();
         });
 
+        QUnit.test('form: quick create for field that returns false after name_create call', async function (assert) {
+            assert.expect(3);
+
+            var prom = testUtils.makeTestPromise();
+            var form = await createView({
+                View: FormView,
+                model: 'partner',
+                data: this.data,
+                arch: '<form>' +
```

Comment:
as it is short:
`'<form><field name="trululu"/></form>'`
otherwise, you can use template strings